3-using-webhook-endpoints/delay/main.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+from pydantic import BaseModel
+import time
+
+class Item(BaseModel):
+    # Add your input parameters here
+    delayMins: Optional[int] = 3
+
+### Setup your model here. 
+"""
+This code will be run once when your replica starts. 
+Load your model, setup datasets, pull in secrets, etc. here
+"""
+
+
+def predict(item, run_id, logger):
+    params = Item(**item)
+
+    for i in range(params.delayMins*10):
```

Comment:
remove the *10. Webhook can still be useful even if short running task. Also its just an example so doesn't need to be precise just show value